GitUI/UserControls/FileStatusDiffCalculator.cs

```diff
@@ -30,46 +51,57 @@ public IReadOnlyList<FileStatusWithDescription> SetDiffs(IReadOnlyList<GitRevisi
             List<FileStatusWithDescription> fileStatusDescs = new();
             if (revisions!.Count == 1)
             {
-                if (selectedRev.ParentIds is null || selectedRev.ParentIds.Count == 0)
+                if (parentsAreRewritten && !selectedRev.IsArtificial)
                 {
-                    Validates.NotNull(selectedRev.TreeGuid);
-
-                    // No parent for the initial commit
-                    fileStatusDescs.Add(new FileStatusWithDescription(
-                        firstRev: null,
-                        secondRev: selectedRev,
-                        summary: GetDescriptionForRevision(describeRevision, selectedRev.ObjectId),
-                        statuses: module.GetTreeFiles(selectedRev.TreeGuid, full: true)));
+                    return PresentSingleRevision(module.GetParents(selectedRev.ObjectId).ToList());
                 }
                 else
                 {
```

Comment:
Nit: The `else` can be omitted after early exit:
```suggestion
                    return PresentSingleRevision(module.GetParents(selectedRev.ObjectId).ToList());
                }
```

Reply:
ok